demos/coreMQTT/mqtt_demo_mutual_auth.c

```diff
@@ -1191,6 +1191,8 @@ static MQTTStatus_t prvWaitForPacket( MQTTContext_t * pxMQTTContext,
         /* Event callback will set #usPacketTypeReceived when receiving appropriate packet. This
          * will wait for at most mqttexamplePROCESS_LOOP_TIMEOUT_MS. */
         xMQTTStatus = MQTT_ProcessLoop( pxMQTTContext, mqttexamplePROCESS_LOOP_TIMEOUT_MS );
+
+        vTaskDelay( mqttexampleDELAY_BETWEEN_PUBLISHES_TICKS );
```

Comment:
What issue does adding the delay cause in CI? It will be valuable to add the a comment for the reasoning